File: rmatnas/src/app/Classes/Family.ts
export class Family {
    Id: number;
    FirstNameFather: string;
    FirstNameMother: string;
    LastName: string;
    Telephone: string;
    PelephoneFather: string;
    PelephoneMother: string;
    Address: string;
    // Email: string;
    Status: string;
    NumChildren: number;
    Reference: string;
    Reason: string;
    show: boolean;
    color: boolean;
    // tslint:disable-next-line: max-line-length
    constructor(fatherName: string, motherName: string, lastName: string, telephone: string, pfather: string, pmother: string, address: string, status: string, nChildren: number, reason: string, reference: string) {
        this.FirstNameFather = fatherName;
        this.FirstNameMother = motherName;
        this.LastName = lastName;
        this.Telephone = telephone;
        this.PelephoneFather = pfather;
        this.PelephoneMother = pmother;
        this.Address = address;
        // this.Email = email;
        this.Status = status;
        this.NumChildren = nChildren;
        this.Reference = reference;
        this.Reason = reason;
        this.show = false;
    }
}

File: Matnas/Common/Family.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Common
{
    public class Family
    {
        public int Id { get; set; }
        public string FirstNameFather { get; set; }
        public string FirstNameMother { get; set; }
        public string LastName { get; set; }
        public string Telephone { get; set; }
        public string PelephoneFather { get; set; }
        public string PelephoneMother { get; set; }
        public string Address { get; set; }
        public string Status { get; set; }
        public int NumChildren { get; set; }
        public string Reference { get; set; }
        public string Reason { get; set; }
    }
}
